x/zkism/types/ism.go

```diff
@@ -13,10 +13,10 @@ const (
 	InterchainSecurityModuleTypeZKExecution = 42
```

Comment:
We should rename this field... doesn't really matter what its called, on that it doesn't collide with the uint8 numbers used by hyperlane's core ism types. https://github.com/bcp-innovations/hyperlane-cosmos/blob/main/x/core/01_interchain_security/types/types.go#L33

We could literally just call it `ModuleType` if we wanted. I'm fine with that or `ModuleTypeZkISM`